src/containers/Explore.js

```diff
@@ -1,9 +1,20 @@
 import React from 'react';
+import {ExploreBox} from '../components/explorebox/index.js';
 
 export const Explore = () => {
     return(
         <main>
             <h1>Explore</h1>
+            <ExploreBox 
```

Comment:
these props look awesome and they are exactly what I was imagining! You can probably assume that the tags property will contain an array of tag names